components/director/internal/graphql/scalars/clob.go

```diff
@@ -0,0 +1,26 @@
+package scalars
+
+import (
+	"io"
+	"log"
+)
+
+type CLOB string
```

Comment:
I think we should store CLOB as []byte.

Reply:
Hmm, what is the reason of that?

Reply:
For me, it is more natural to store some unstructured, possibly binary data in []byte. You can easily then convert to a string if needed.

Reply:
changed to []byte